eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -32,6 +32,7 @@ library
   exposed-modules:
     Cardano.Ledger.Conway
     Cardano.Ledger.Conway.Core
+    Cardano.Ledger.Conway.Era
```

Comment:
Please, don't change visibility of modules without serious consideration!
This is a very dangerous change, because it would leak all of the orphan instances from the package as orphans!
Revert it back and instead re-export those definitions from `Cardano.Ledger.Conway`
```suggestion
```